src/compiler/build/write-build.ts

```diff
@@ -21,11 +31,10 @@ export const writeBuild = async (config: d.Config, compilerCtx: d.CompilerCtx, b
 
     // successful write
     // kick off writing the cached file stuff
-    // await compilerCtx.cache.commit();
     buildCtx.debug(`in-memory-fs: ${compilerCtx.fs.getMemoryStats()}`);
-    // buildCtx.debug(`cache: ${compilerCtx.cache.getMemoryStats()}`);
 
-    await outputServiceWorkers(config, buildCtx), await validateBuildFiles(config, compilerCtx, buildCtx);
+    await outputServiceWorkers(config, buildCtx);
+    await validateBuildFiles(config, compilerCtx, buildCtx);
```

Comment:
The comma operator here wasn't doing anything...removed for clarity

Reply:
Do these need to run synchronously? Or could we `Promise.all()` them?

Reply:
I don't know if it makes sense to run them in parallel, as the latter there should be validating build output (ie the build should have been completed (including SW gen) before we try to verify anything)